### DISCUSSION TITLE
阿里云盘 API 又被限速了？

### DISCUSSION THREAD

**Discussion by vctao** [messense/aliyundrive-webdav#978] [Q&A]
### 问题描述

今天用infuse测速，下载速度只有25mbps，正常下载速度是100mbps，阿里云盘 API 又被限速25mbps了？

### 重现步骤

infuse 7pro 挂载 webdav测速

### 版本

2.3.3

### 运行平台

docker

### 日志

现在速度只有25mbps，之前正常速度应该是100mbps左右。

**Comment by MarsWoooWH** [comment]
我也感觉限速了  其他人也这样么？

**Comment by tlze** [comment]
网上看到说限速是针对账号的，不是所有人。

**Reply by MarsWoooWH** [reply]
昨天更新了alist 3.33，没有限速

**Comment by akispho** [comment]
明确感觉限速了 这两天看50g左右电影缓冲非常慢

**Comment by Mjmaster** [comment]
今天彻底限速了，200KB/s，看来必须要买第三方权益包才行了。

**Reply by MarsWoooWH** [reply]
看来是时候抛弃阿里云了

**Reply by hotkinkin** [reply]
我的现在是500k，相当的慢，有其他的替代的出来了吗

**Reply by MarsWoooWH** [reply]
现在非会员都是500k了吧，现在要么转115，要么就等大佬有没有解决方案了。

**Reply by shuaiqidegaga** [reply]
限速到500k,啥都看不了了

**Reply by best-cloud** [reply]
貌似 SVIP 每个月也只有 10GB 的高速流量，超过 10GB 就变成 500k 了

**Reply by Mjmaster** [reply]
是的，SVIP 10G，要买第三方权益包才有1T高速流量，虽然也不太够。
